win32/customactions.cpp

```diff
@@ -137,7 +137,11 @@ MD_REGISTRATION minidriver_registration[] = {
                                           13, {0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff}},
 	{TEXT("CardOS 4.4"),                      {0x3b,0xd2,0x18,0x02,0xc1,0x0a,0x31,0xfe,0x58,0xc8,0x0d,0x51},
                                           12, {0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff}},
-	{TEXT("CardOS v5.0"),                     {0x3b,0xd2,0x18,0x00,0x81,0x31,0xfe,0x58,0xc9,0x01,0x14},
+	{TEXT("CardOS v5.0 (a)"),                 {0x3b,0xd2,0x18,0x00,0x81,0x31,0xfe,0x58,0xc9,0x01,0x14},
+                                          11, {0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff}},
+	{TEXT("CardOS v5.0 (b)"),                 {0x3b,0xd2,0x18,0x00,0x81,0x31,0xfe,0x58,0xc9,0x02,0x17},
+                                          11, {0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff,0xff}},
+	{TEXT("CardOS v5.0 (c)"),                 {0x3b,0xd2,0x18,0x00,0x81,0x31,0xfe,0x58,0xc9,0x03,0x16},
```

Comment:
I would mark these strings as CardOS 5.3 as initially marked in the cardos driver since these are really a new version (even though the `card->type` is the same since there was no difference in handling):

https://github.com/OpenSC/OpenSC/blob/master/src/libopensc/card-cardos.c#L54

For me, this is the first time I hear about this file so if we want to have the consistency between the ATRs in drivers and in md, I think we should write a simple script to check whether they match and update it globally, not just opportunistically when somebody notices.